web/app_options.js

```diff
@@ -133,6 +133,14 @@ const defaultOptions = {
     value: -1,
     kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
   },
+  standardFontDataUrl: {
+    /** @type {string} */
+    value:
+      typeof PDFJSDev === "undefined" || !PDFJSDev.test("PRODUCTION")
+        ? "../external/standard_fonts/"
+        : "../web/standard_fonts/",
+    kind: OptionKind.API,
+  },
```

Comment:
Given that this uses `OptionKind.API`, and that the different kinds are grouped together, please move this option down to just after this line instead: https://github.com/mozilla/pdf.js/blob/239d0097fa5c717ba8dde56e175ae006c9053df1/web/app_options.js#L219